dev.dart

```diff
@@ -37,4 +37,9 @@ void main({List<String> arguments}) async {
   var newConfig = {...baseConfig, ...projConfig[projName]};
   var newFile = File('web/assets/constants.json');
   newFile.writeAsStringSync(jsonEncode(newConfig));
+
+  print('');
+  print('When using webdev serve, open http://localhost:8080/ in your browser.');
+  print('      .. 127.0.0.1 does not work for this app');
```

Comment:
hehe yes, I think it needs to be enabled somewhere in firebase..

I always go to localhost by default, I hardly pay attention to webdev serve's URL (perhaps EB does the same) - hope you didn't spend too much time debugging!